x-pack/plugins/timelines/public/components/t_grid/body/helpers.test.tsx

```diff
@@ -138,7 +138,7 @@ describe('helpers', () => {
       ]);
     });
 
-    test('it defaults to a `columnType` of empty string when a column does NOT has a corresponding entry in `columnHeaders`', () => {
+    test('it defaults to a `columnType` of empty string when a column does NOT have a corresponding entry in `columnHeaders`', () => {
```

Comment:
@janmonschke this is the follow-up from our [previous conversation](https://github.com/elastic/kibana/pull/132190#discussion_r874954369)

Reply:
It's beautiful 😍